pingcap/tiflash
# refine LAC in disaggregated mode

### What problem does this PR solve?

Issue Number: close #8178

Problem Summary:
1. Do not start LAC and pipeline TaskScheduler for **WN**. Because WN will not run any queires. And for now WN doesn't support resource control.
2. Make low token threshold lower. 0.8 is too high. It may cause rpc to GAC too frequent.
3. Delete server id from GAC when LAC stopped. Because on Serverless Tier, **CN** will scale in/out frequently, so we can avoid too many server id allocated from GAC.

### What is changed and how it works?

### Check List

Tests 

1. start tidb cluster with one or more tiflash
2. search log(keyword: get unique_client_id succeed) to check the unique client id of LAC. Run `etcdctl get --prefix '/tidb/server_id/{client_id_of_LAC}'` to check it from GAC, should get successfully.
4. kill tiflash
5. run `etcdctl get --prefix '/tidb/server_id/{client_id_of_LAC}'` , the server id has been deleted.

Side effects


Documentation


### Release note



```release-note
None
```
